LanguageMaster.xml

```diff
@@ -1421,8 +1421,10 @@ can edit in same subtitle file (collaboration)</Information>
     <NewLine>New line (\r\n)</NewLine>
     <NewLineShort>New line (\n)</NewLineShort>
     <AnyDigit>Any digit (\d)</AnyDigit>
+    <NonDigit>Non digit (\D)</NonDigit>
     <AnyCharacter>Any character (.)</AnyCharacter>
     <AnyWhitespace>Any whitespace (\s)</AnyWhitespace>
+    <NonSpaceCharacter>Non space character (\S)</NonSpaceCharacter>
```

Comment:
Space is used for ASCII 32 (U+0020).
Whitespace[1] is used for a _set_ of characters, including space, nbsp, tab, linefeed, and others.
`\s` represents the set of all white space characters, `\S` is its complement (the set of characters that are not in `\s`).

[1] Actually, two words "white space", but "whitespace" seems to be more commonly used these days.